components/atom/button/README.md

```diff
@@ -0,0 +1,39 @@
+# AtomButton
+
+> Atom Element: SUI button
+
+![](./assets/screenshot.png)
+
+## Installation
+
+```sh
+npm install @schibstedspain/sui-atom-button --save
```

Comment:
I thought that our sui-components would be under @s-ui

Reply:
No. Only tools. Not components.